gcloud/datastore/helpers.py

```diff
@@ -218,44 +218,6 @@ def _get_value_from_property_pb(property_pb):
     return _get_value_from_value_pb(property_pb.value)
 
 
-def _set_protobuf_property(property_pb, name, value, indexed):
```

Comment:
It wasn't being used, and showed up as dead code.  It was already basically folded into [assign_entity_to_mutation](https://github.com/GoogleCloudPlatform/gcloud-python/blob/64aaed4190ca4b0b9457105636d0ec6893e824ca/gcloud/datastore/batch.py#L276), which would now be its only caller (I think).